### PR TITLE
linked to backend with alamofire, treated the get request for data and the post to treat the notification's adding 

### PR DESCRIPTION
**Pull request by alaaDouzi**

### CODE REVIEW

File: ios/resevation/ViewsControllers/ViewController.swift [hunk 1]
```diff
@@ -146,24 +167,46 @@ class ViewController: UIViewController,UNUserNotificationCenterDelegate {
...
                     self.notifIcon.isHidden = false
                     //let userActions = "User Actions"
-                    
+
                     let date = parseDateToString(date: dateReservation) + parseTimeToString(date: dateReservation)
                     self.ajouterNotif(dateNotif: self.dateNotif, dateReservation: date, rider: self.ride.passenger.fullName, identifier:self.ride.id)
-                    
-                }else{
+
+                } else if response.response?.statusCode == 400{
```
Comment (Moumene) [inline ios/resevation/ViewsControllers/ViewController.swift]: very nice 👍

File: ios/resevation/ViewsControllers/ViewController.swift [hunk 2]
```diff
@@ -146,24 +167,46 @@ class ViewController: UIViewController,UNUserNotificationCenterDelegate {
...
                     print("alerte indiquant l'entrée dune date non valide")
                     let alert = UIAlertController(title: "Date erronnée", message: "Vous avez introduit une date passée", preferredStyle: .alert)
                     alert.addAction(UIAlertAction(title: "OK", style: .cancel, handler:nil))
                     self.present(alert, animated: true)
+                } else {
+                    print("don't know what the fuck u just did !!")
```
Comment (Moumene) [inline ios/resevation/ViewsControllers/ViewController.swift]: lol

**Review by Moumene — APPROVED** [review]
👍

**Comment by Moumene** [comment]
I suggest you update the PR title to something clear like describing the purpose from your changes or the issue you are fixing etc...

**Comment by Moumene** [comment]
Better like this 👍

**Comment by Alaa1908** [comment]
persisted the data using realm database, implement the feature of cash in case of non connection to the server, updated the README to show the steps of installing the IOS project

**Pull request merged**